### DISCUSSION TITLE
[turborepo] NPM RUN DEV with Turborepo, the local server gets stuck on "COMPILING" indefinitely.

### DISCUSSION THREAD

**Discussion by sahiwl** [vercel/next.js#68443] [Help]
### Summary

while learning turborepo, i bootstrapped turborepo using `npm create-turbo@latest`, which caused [this](https://github.com/vercel/turbo/issues/4183) error. As suggested [here](https://github.com/vercel/turbo/issues/4183#issuecomment-1668737319), which helped a lot of users to atleast run the local dev server, I did ran `npx next telemetry disable` which resolved the error, but now `npm run dev` does get stuck on **Compiling** and proceeds to give nothing in output.

For more info, i [commented](https://github.com/vercel/turbo/issues/4183#issuecomment-2265178263) under that issue for the same. 

![image](https://github.com/user-attachments/assets/bac04784-232e-42a9-b602-80c18e3db998)

### Additional information

```js
"packageManager": "npm@10.2.4", 
"next": "15.0.0-rc.0",
"turbo": "^2.0.11"
```

**Comment by gethari** [comment]
did you get solution to this ?

**Reply by sahiwl** [reply]
[https://github.com/vercel/turborepo/issue"s/4183#issuecomment-2290837160](https://github.com/vercel/turborepo/issues/4183#issuecomment-2290788567)
check this thread out.
